kong/pdk/vault.lua

```diff
@@ -1429,8 +1429,6 @@ local function new(self)
       end
     end
 
-    LRU:flush_all()
-
```

Comment:
The `LuaDoc` has:

```
  ---
  -- Flushes LRU caches and forcibly rotates the secrets.
  --
  -- This is only ever executed on traditional nodes.
  ```

I guess this may need to be changed. Also should it be executed with incremental too?